6.5.remember.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from PIL import Image
+
+
+def img_encode(txt, path = None):
+    matrix = np.array([[255] * len(txt) for _ in range(ord('z'))])
+
+    for i in range(len(txt)):
```

Comment:
rename 'i' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing